### PR TITLE
Added migration sql script: create project.templateId and new tables

### PR DESCRIPTION
**Pull request by ngoctay**
@coderReview

### CODE REVIEW

File: migrations/20180608_project_add_templateId_and_new_tables.sql [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+--   project_templates
+--   project_types
+--   timelines
+--
+
+--
+-- projects
+--
+ALTER TABLE projects ADD COLUMN "templateId" bigint;
+
```
[the remark this answers is not in the document]
Comment (ngoctay) [reply migrations/20180608_project_add_templateId_and_new_tables.sql]: I am not sure about that column.
I found it in another script: https://github.com/topcoder-platform/tc-project-service/blob/feature/dev-challenges/migrations/project_add_version_column.sql

[the remark this answers is not in the document]
Comment (ngoctay) [reply migrations/20180608_project_add_templateId_and_new_tables.sql]: Do you want to remove "project_add_version_column.sql", and add the "version" column into the new script?

If so, what is the default value? v2 or v3?

[the remark this answers is not in the document]
Comment (coderReview) [reply migrations/20180608_project_add_templateId_and_new_tables.sql]: @vikasrohit is it ok to have two scripts to run on prod?

- 20180608_project_add_templateId_and_new_tables.sql 

- project_add_version_column.sql

[the remark this answers is not in the document]
Comment (vikasrohit) [reply migrations/20180608_project_add_templateId_and_new_tables.sql]: Although it is okay to have multiple scripts, I would like single script for single release.

[the remark this answers is not in the document]
Comment (coderReview) [reply migrations/20180608_project_add_templateId_and_new_tables.sql]: @ngoctay when running the scripts old records should have v2 values.

[the remark this answers is not in the document]
Comment (coderReview) [reply migrations/20180608_project_add_templateId_and_new_tables.sql]: @ngoctay I think we can merge those 2 scripts into one.

[the remark this answers is not in the document]
Comment (ngoctay) [reply migrations/20180608_project_add_templateId_and_new_tables.sql]: done
https://github.com/topcoder-platform/tc-project-service/pull/91/commits/56cafd18165fcf091242917088b8e515af2b9cc1

**Pull request merged**